src/Containers/AddressVerifier/AddressVerifier.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react'
+import { getValidatedAddress } from '../../utils/api'
+
+const AddressVerifier = () => {
+  const [addressLines, setAddressLines] = useState([]);
```

Comment:
any specific reasons why it's array?

Reply:
because the Google API requires entered address is in an array